jujugui/static/gui/src/app/components/terminal/terminal.js

```diff
@@ -78,17 +83,58 @@ class Terminal extends React.Component {
       }
       // Terminado sends a "disconnect" message when the process it's running
```

Comment:
It seems that this comment should now be moved down inside the disconnect case.

Reply:
done